src/player.js

```diff
@@ -1508,6 +1503,14 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  _handleFrameDrop(event: FakeEvent): void {
+    const sortedVideoTracks = this._getTracksByType(TrackType.VIDEO).sort((trackA, trackB) => trackA.bandwidth > trackB.bandwidth);
+    const currentBandwidthTrackIndex = sortedVideoTracks.findIndex(track => track.bandwidth === event.payload.bandwidth);
+    if (currentBandwidthTrackIndex - 1 >= 0) {
```

Comment:
findIndex returns -1 in case of no matching track.
`currentBandwidthTrackIndex !== -1` will be more readable condition.